src/editors/containers/VideoEditor/index.jsx

```diff
@@ -24,11 +29,22 @@ export default function VideoEditor({
       </div>
     </EditorContainer>
   );
-}
+};
 
 VideoEditor.defaultProps = {
   onClose: null,
+  videoSettings: null,
 };
 VideoEditor.propTypes = {
   onClose: PropTypes.func,
+  // redux
+  videoSettings: PropTypes.shape({}),
```

Comment:
you can have two different shapes here (loaded/unloaded)

Reply:
This covers both loaded and unloaded

Reply:
Sorry, what I meant was, this does pass, but gives no information to the reader what kind of prop this is, instead try:

`videoSettings: PropTypes.oneOfType([Proptypes.shape({}), Proptypes.Shape({key1: Proptypes.something, key2... })])`

Reply:
Oh I see. I updated it to reflect what is in mockVideoData